src/poetry/console/commands/source/add.py

```diff
@@ -50,8 +69,30 @@ def handle(self) -> int:
             )
             return 1
 
+        if is_default or is_secondary:
+            if priority is not None:
+                self.line_error(
+                    "Priority was passed through both --priority and a deprecated flag"
+                    " (--default or --secondary). Please only provide one of these."
```

Comment:
```suggestion
                    "<error>Priority was passed through both --priority and a deprecated flag"
                    " (--default or --secondary). Please only provide one of these.</error>"
```
(red color)

Reply:
Should this then also start with "Error", in line with https://github.com/python-poetry/poetry/pull/7658/files#r1155309417? I do note that other occurrences of `self.line_error("<error>...` do also not start with "Error:", but I don't know whether that is on purpose.

Reply:
We are not very consistent about that. You always have to consider that you can't rely just on colors. It's probably good practice to start with `"Error:"` (at least if it's not absolutely clear from the message itself).

Reply:
On the other hand, the text of uncaught exceptions is only printed in red without any prefix. Thus, I'd say `Error:` or not should be clarified in general and that's something for another PR.